### DISCUSSION TITLE
where does it get the entity_ids from?

### DISCUSSION THREAD

**Discussion by commentsonly** [mak-gitdev/HA_enoceanmqtt#9] [Q&A]
After everything works ok I do wonder how the names of buttons are built.
For example my Gira got the entries
e2m_  <=  likely comes from the addon +
button_1 <= this is what I named it in the /config/enoceanmqtt.devices file +
_ao_pressed
_ai_pressed
_bo_pressed
_bi_pressed
top left, bottom left, top_right and bottom_right would make more sense to me.
I know how to get this solved .. one must simply rename the entries in
/config/.storage/core.entity_registry file
or have I missed something vital in HA throughout the years how this could be done much easier?
Modifying the friendly name is lots easier but that's not what I want to do

**Comment by mak-gitdev** [comment]
AO, AI, BO, BI are the official names of these buttons in the EnOcean EEP documentation.
Top left in your case may mean bottom right for another device. But AI, AO, BO, BI will always be the same.

So I can only recommend to change the friendly name :)
